library/Vanilla/EmbeddedContent/Factories/TwitchEmbedFactory.php

```diff
@@ -118,7 +118,7 @@ private function idFromUrl(string $url): ?string {
             return null;
         }
 
-        if ($host === "clips.twitch.tv" || preg_match("`/(?<channel>[^/]+)/clip/(?<clipID>[^/]+)`", $path, $clipsMatch)) {
+        if ($host === "clips.twitch.tv" && preg_match("`/(?<channel>[^/]+)/clip/(?<clipID>[^/]+)`", $path, $clipsMatch)) {
```

Comment:
This is one of the only material code changes, but I can't see how it wasn't a bug before.